File: 344-reverse-string/reverse-string.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        n = len(s)
        start = 0
        e = n-1
        while(start<e):
            temp = s[start]
            s[start]=s[e]
            s[e]=temp
            start+=1
            e-=1

        

File: 344-reverse-string/reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        int n = s.length;
        int start = 0;
        int e = n-1;
        char temp = 'a';
        while(start<e){
            temp = s[start];
            s[start]=s[e];
            s[e]=temp;
            start++;
            e--;
        }
        
    }
}